bin/git-forgit

```diff
@@ -1002,6 +1003,95 @@ _forgit_ignore_clean() {
     [[ -d "$FORGIT_GI_REPO_LOCAL" ]] && rm -rf "$FORGIT_GI_REPO_LOCAL"
 }
 
+_forgit_worktree_preview() {
+    local sha
+    # trailing space in  grep to avoid matching worktrees with a common path
+    sha=$(git worktree list | grep "$1 " | awk '{print $2}')
+    [[ "$sha" == "(bare)" ]] && return
+    # the trailing '--' ensures that this works for branches that have a name
+    # that is identical to a file
+    git log "$sha" "${_forgit_log_preview_options[@]}" --
```

Comment:
- I should probably add preview options `FORGIT_WORKTREE_PREVIEW_GIT_OPTS` (like #396)
- I've noticed this can run a bit slow if you have a repo with a long history because it attempts to get the entire history for the branch checked out in that worktree
- Would be quicker if I limit the log entries in the worktree preview options